packages/terra-button-group/src/terra-dev-site/doc/button-group/ChangeLog.3.doc.mdx

```diff
@@ -0,0 +1,7 @@
+import { Badge } from 'terra-button-group/package.json?dev-site-package';
+
+import ChangeLog from '../../../../CHANGELOG.md';
```

Comment:
You can and should use the package name instead of traversing the folder structure (../../../) for ALL of your imports, not just the badge.
```suggestion
import ChangeLog from 'terra-button-group/CHANGELOG.md';
```

Reply:
updated here https://github.com/cerner/terra-core/pull/2815/commits/994a492f7ec5a562f67aa3285a2b49bf117cef0e.